packages/compiler/core/program.ts

```diff
@@ -221,7 +221,7 @@ export function projectProgram(
   return createProjector(program, projections, startNode);
 }
 
-export async function createProgram(
+export async function compile(
```

Comment:
I think server still uses createProgram, which I expected to see changed here, and annoyingly has its own `compile` that wraps it. But I guess we could find another name in server or `import { compile as compileCore }`.